packages/p2p/src/components/modal-manager/modals/my-ads-delete-error-modal/my-ads-delete-error-modal.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { Button, Modal, Text } from '@deriv/components';
+import { isMobile } from '@deriv/shared';
+import { observer } from '@deriv/stores';
+import { Localize } from 'Components/i18next';
+import { useModalManagerContext } from 'Components/modal-manager/modal-manager-context';
+import { useStores } from 'Stores';
+
+const MyAdsDeleteErrorModal = () => {
+    const { hideModal, is_modal_open } = useModalManagerContext();
+    const { my_ads_store } = useStores();
+    const { delete_error_message } = my_ads_store;
+
+    return (
+        <Modal
+            className='delete-modal'
+            has_close_icon={false}
+            is_open={is_modal_open}
+            renderTitle={() => (
+                <Text color='prominent' size={isMobile() ? 'xs' : 's'} weight='bold'>
+                    <Localize i18n_default_text='Do you want to delete this ad?' />
+                </Text>
+            )}
+            width='440px'
+        >
+            <Modal.Body>{delete_error_message}</Modal.Body>
+            <Modal.Footer>
+                <Button.Group>
+                    <Button primary large onClick={() => hideModal()}>
```

Comment:
```suggestion
                    <Button primary large onClick={hideModal}>
```

Reply:
It throws this error as adrienne mentioned in his modal manager refactoring PR.
<img width="911" alt="Screenshot 2023-07-27 at 10 12 31 AM" src="https://github.com/farrah-deriv/deriv-app/assets/125247833/0aba7fc4-3b12-4e64-aa4c-7df3dbc5e59a">